emails/models.py

```diff
@@ -116,6 +116,7 @@ class Profile(models.Model):
     remove_level_one_email_trackers = models.BooleanField(null=True, default=False)
     onboarding_state = models.PositiveIntegerField(default=0)
     auto_block_spam = models.BooleanField(default=False)
+    forwarded_first_reply = models.BooleanField(default=False, blank=True, null=True)
```

Comment:
Yup, backward-compatibility. I forgot if there's a down-side to this approach? And maybe I was just lazy trying to avoid writing another `add_db_default_forward_func`?

Reply:
Thanks for the explanation. The only down-side is that the code here is a bit wonky. It's default to `False` but accepts `null` or `blank` value which is why we were using the `add_db_default_forward_func`. I think it's fine as is do it this way and would be even better if we have a follow-up to remove the `blank` and `null` values since it would make the data check easier. Let's discuss this with all backend engineers to see if we can set a standard for this problem?